Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM ubuntu:18.04
```

Comment:
We should be using something like `alpine`, `golang:alpine` or `gcr.io/distroless/static-debian11` image.  This will result in a smaller image and more secure

Reply:
Unfortunately, there is no way to install or build MKL on Alpine. MKL is required for building Bergamot (translation engine we use). For now `builder` is on `golang:1.16` and `artifact` is on `ubuntu:20.04`.